src/scripts/site.js

```diff
@@ -316,20 +316,6 @@ var AppInfo = {};
     }
 
     function getSettingsBuilder(UserSettings, layoutManager, browser) {
```

Comment:
can we just get rid of this function?..

Reply:
It would take a lot of refactoring and I didn't want to break too much in `site.js` while @vitorsemeano is spending so much time in that file.